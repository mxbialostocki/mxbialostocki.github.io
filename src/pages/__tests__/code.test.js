import React from "react"
import ShallowRenderer from "react-test-renderer/shallow"

import Code from "../code"
import { useStaticQuery } from "gatsby"

beforeAll(() => {
  const childImageSharp = {
    fluid: {
      base64: "base64",
      aspectRatio: "aspectRatio",
      src: "src",
      srcSet: "srcSet",
      sizes: "sizes"
    }
  }
  useStaticQuery.mockReturnValue({
    react: {
      childImageSharp
    },
    graphqlicon: {
      childImageSharp
    },
    nodejs: {
      childImageSharp
    },
    gatsby: {
      childImageSharp
    },
    materialui: {
      childImageSharp
    },
    mongodb: {
      childImageSharp
    },
    github: {
      childImageSharp
    }
  })
})

describe("Code Page", () => {
  const renderer = new ShallowRenderer()
  it("renders correctly", () => {
    const tree = renderer.render(
      <Code/>
    )
    expect(tree).toMatchSnapshot()
  })
})