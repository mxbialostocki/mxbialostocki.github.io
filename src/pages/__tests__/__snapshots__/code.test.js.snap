// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code Page renders correctly 1`] = `
<React.Fragment>
  <Layout>
    <SEO
      description=""
      lang="en"
      meta={Array []}
      title="code"
    />
    <WithStyles(ForwardRef(Grid))
      container={true}
      direction="row"
      spacing={4}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
        md={11}
      >
        <WithStyles(ForwardRef(Typography))
          variant="h5"
        >
          I'm a full-stack dev with a current focus on web development. A few projects I've worked on lately are below. I'm always open to new challenges, so please get in touch with me if you'd like to work together.
        </WithStyles(ForwardRef(Typography))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
    <Hidden
      only={
        Array [
          "xs",
          "sm",
          "md",
        ]
      }
    >
      <WithStyles(ForwardRef(Grid))
        alignItems="center"
        container={true}
        direction="row"
        justify="center"
        spacing={6}
        style={
          Object {
            "padding": "1rem 0",
          }
        }
      >
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={1}
        >
          <Image
            Tag="div"
            alt=""
            className="makeStyles-stack-12"
            durationFadeIn={500}
            fadeIn={true}
            fluid={
              Object {
                "aspectRatio": "aspectRatio",
                "base64": "base64",
                "sizes": "sizes",
                "src": "src",
                "srcSet": "srcSet",
              }
            }
            loading="lazy"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={1}
        >
          <Image
            Tag="div"
            alt=""
            className="makeStyles-stack-12"
            durationFadeIn={500}
            fadeIn={true}
            fluid={
              Object {
                "aspectRatio": "aspectRatio",
                "base64": "base64",
                "sizes": "sizes",
                "src": "src",
                "srcSet": "srcSet",
              }
            }
            loading="lazy"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={1}
        >
          <Image
            Tag="div"
            alt=""
            className="makeStyles-stack-12"
            durationFadeIn={500}
            fadeIn={true}
            fluid={
              Object {
                "aspectRatio": "aspectRatio",
                "base64": "base64",
                "sizes": "sizes",
                "src": "src",
                "srcSet": "srcSet",
              }
            }
            loading="lazy"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={1}
        >
          <Image
            Tag="div"
            alt=""
            className="makeStyles-stack-12"
            durationFadeIn={500}
            fadeIn={true}
            fluid={
              Object {
                "aspectRatio": "aspectRatio",
                "base64": "base64",
                "sizes": "sizes",
                "src": "src",
                "srcSet": "srcSet",
              }
            }
            loading="lazy"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={1}
        >
          <Image
            Tag="div"
            alt=""
            className="makeStyles-stack-12"
            durationFadeIn={500}
            fadeIn={true}
            fluid={
              Object {
                "aspectRatio": "aspectRatio",
                "base64": "base64",
                "sizes": "sizes",
                "src": "src",
                "srcSet": "srcSet",
              }
            }
            loading="lazy"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={1}
        >
          <Image
            Tag="div"
            alt=""
            className="makeStyles-stack-12"
            durationFadeIn={500}
            fadeIn={true}
            fluid={
              Object {
                "aspectRatio": "aspectRatio",
                "base64": "base64",
                "sizes": "sizes",
                "src": "src",
                "srcSet": "srcSet",
              }
            }
            loading="lazy"
          />
        </WithStyles(ForwardRef(Grid))>
        <WithStyles(ForwardRef(Grid))
          item={true}
          xs={1}
        >
          <Image
            Tag="div"
            alt=""
            className="makeStyles-stack-12"
            durationFadeIn={500}
            fadeIn={true}
            fluid={
              Object {
                "aspectRatio": "aspectRatio",
                "base64": "base64",
                "sizes": "sizes",
                "src": "src",
                "srcSet": "srcSet",
              }
            }
            loading="lazy"
          />
        </WithStyles(ForwardRef(Grid))>
      </WithStyles(ForwardRef(Grid))>
    </Hidden>
    <WithStyles(ForwardRef(Grid))
      container={true}
      direction="row"
      spacing={4}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
        sm={12}
      >
        <Projects />
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
  </Layout>
</React.Fragment>
`;
